spec/draper/query_methods_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+module Draper
+  describe QueryMethods do
+    describe "#method_missing" do
```

Comment:
I think you forgot the single quotes on this one :nail_care:

Reply:
Good catch, thanks.